src/config.py

```diff
@@ -0,0 +1,23 @@
+'''
+Used by __init__.py to set the MongoDB configuration for the whole app
+The app can then be references/imported by other script using the __init__.py
+'''
+
+DEBUG = True
+TESTING = True
+
+# mongo db
+SECRET_KEY = "vusualyzerrrrrrrr"
```

Comment:
Should a secret key be part of git?

Reply:
See: https://github.com/UMCUGenetics/ExonCov/blob/9fab496a41c6e28ff22e5a193ea32735d39373bf/config.py#L16 

I randomly generate one for each server we checkout ExonCov on.

Reply:
So indeed your production key should not be on github.

Reply:
We talked about this in teams. I searched (partly) wrong. It is a key from flask [link](https://flask.palletsprojects.com/en/2.1.x/config/#SECRET_KEY).
Should not be on Github. I will change this to `SECRET_KEY = ""` and give an explanation in the Readme to change this. if not filled in by someone, it will generate an error.

If someone changes the key it is not a big deal. cookie sessions will reset and people need to login again.
The secret key can be used to generate fake cookies, to gain access. therefore I will remove it from Github and make sure that the key is always something random.